extensions/eclipse-che-theia-git-provisioner/src/node/git-configuration-controller.ts

```diff
@@ -61,8 +61,8 @@ export class GitConfigurationController {
                     const userConfig = await this.getUserConfigurationFromGitConfig();
                     const preferences = await this.preferencesService.getPreferences();
 
-                    preferences[GIT_USER_NAME] = userConfig.name;
-                    preferences[GIT_USER_EMAIL] = userConfig.email;
+                    (preferences as { [index: string]: string })[GIT_USER_NAME] = userConfig.name!;
```

Comment:
Maybe create/reuse separate type for `[index: string]: string` ? See [here](https://github.com/eclipse/che-theia/blob/master/extensions/eclipse-che-theia-plugin/src/che-proposed.d.ts#L20)